setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 ignore = 
     E203,
+    E265, # Throws errors for '#%%' delimiter in VSCode jupyter notebook syntax
```

Comment:
Let's remove this and add `# noqa: E265` on the errored lines

Reply:
That would be super annoying, as this delimiter is the equivalent of starting a new cell in Jupyter...

Reply:
The problem is these are still just `.py` files, so can't do any kind of per-file ignore (which is an option in flake).  Unless there is a *strong* objection I'm going to leave this in- I think it's a really nice function in VSC, but more importantly, potentially a really nice way to store and version our jupyter notebooks going forwards

Reply:
Ok that's fine